janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java

```diff
@@ -185,6 +186,13 @@ public CQLStoreManager(final Configuration configuration) throws BackendExceptio
         fb.optimisticLocking(true);
         fb.multiQuery(false);
 
+        Boolean externalLocking = configuration.get(EXTERNAL_LOCKING);
+        LOGGER.info("External locking: {}", externalLocking);
```

Comment:
I don't think this log is necessary, please remove.

Reply:
Done